src/modules/sensors/sensor_params.c

```diff
@@ -3543,3 +3543,18 @@ PARAM_DEFINE_FLOAT(RC_FLT_SMP_RATE, 50.0f);
  * @group Radio Calibration
  */
 PARAM_DEFINE_FLOAT(RC_FLT_CUTOFF, 10.0f);
+
+/**
+* Driver level cut frequency for gyro/accel
+*
+* The cut frequency for the 2nd order butterworth filter on the imu driver. This features
+* is currently supported by the mpu6000 and mpu9250. This only affects the signal sent to the
+* controllers, not the estimators. 0 disables the filter.
+*
+* @min 5
+* @max 30
```

Comment:
I don't see the checks in the code enforcing min/max
why 30 ? AFAIK @priseborough was suggesting 67 Hz.